reporter/iface.go

```diff
@@ -87,10 +87,16 @@ type SymbolReporter interface {
 	// open the file and then enqueue the upload in the background.
 	ExecutableMetadata(args *ExecutableMetadataArgs)
 
+	// FrameKnown may be used to query the reporter if the FrameID is known. This is mainly
+	// used to see if the data is already cached and extra work to resolve the metadata
+	// should not be done. If the reporter returns false, the interpreter plugins will
```

Comment:
```suggestion
	// used to see if the data is already cached and to avoid extra work resolving the metadata.
	// If the reporter returns false, the interpreter plugins will
```